package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "lerna run test",
+    "clean": "lerna exec npm run clean",
```

Comment:
this is best done through `lerna run clean`, it's almost the same but I think the logging output is better.